pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/creator/CombinedInvertedIndexCreator.java

```diff
@@ -18,6 +18,80 @@
  */
 package org.apache.pinot.segment.spi.index.creator;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.apache.pinot.spi.data.FieldSpec;
+
+
+/**
+ * This is the index used to create range indexes
```

Comment:
s/range/inverted

Reply:
why? this is used to create range indexes, right? Maybe it is also used to create other inverted indexes?

Reply:
I think that's the reason why I added the comment, because each time I arrived on this class I didn't remember it was related to range indexes instead of inverted :laughing: